### ISSUE TITLE
Resource Page Idea: How to make an accessible, user-friendly PDF

### ISSUE THREAD

**Issue by jeremyzilar** [GSA/digitalgov.gov#1209]
As much as we'd like to get away from them, PDFs are a common factor of government life. So when we have to make them, they should be the best they can be for everyone (right?). So how does someone go about making a great, accessible, and user-friendly PDF. 

Maybe we can work together to make a resource page on digital.gov that covers the basics and provides some links to helpful resources for getting started?

## Here are some questions to get this started
- What are some resources that people can go to to learn how to create accessible and user-friendly PDFs?
- Are there any checklists for ensuring that the PDF you're about to create, needs to be a PDF?
- What are the qualities of a well-built PDF?
- What software is best for making accessible PDFs?

_This started as a discussion in the [UX Community](https://digital.gov/communities/user-experience/)_

### Help us to shape this page
Please leave comments/ideas/suggestions below 😺

**Comment by jeremyzilar** [comment]
There are a wealth of resources on **Section508.gov** — https://www.section508.gov/create/pdfs

**Comment by JenniferHoran** [comment]
Please coordinate with the AED CoP from the CIO Council, there has been years worth of work done on PDF, Word, and now PPT and Excel. Updated content is posted up on Section508.gov. Nic from DHS OAST and the 508 team at FAA are in the process of finalizing newer video content. We are talking about Word to PDF, mainly, for the layperson (who doesn't even know how to use their tools in the Word ribbon in the first place). Design team members who use InDesign to PDF and Forms Managers who use InDesign to PDF would be very valuable, as we do not have that content fully explored with AED CoP team. https://section508.gov/create
https://www.youtube.com/playlist?list=PLXa30RYdupU52-eIJ-Nd73KwF2asIMdxd
https://section508.gov/create/pdfs

**Comment by jeremyzilar** [comment]
Also, see this checklist from Social Security's Section 508 team: https://www.ssa.gov/accessibility/checklists/pdf/pdfchecklist.html
